root/components/Paginator.js

```diff
@@ -127,19 +127,19 @@ component Paginator(
           </li>
         ) : null}
 
-        <li className="separator" key="separate-next">
-          {nextPage == null ? (
-            <li key="no-next">
-              <span>{l('Next')}</span>
-            </li>
-          ) : (
-            <li key="next">
-              <a href={uriPage(reqUri, pageVar, nextPage, hash)}>
-                {l('Next')}
-              </a>
-            </li>
-          )}
-        </li>
+        <li className="separator" key="separate-next" />
+
+        {nextPage == null ? (
```

Comment:
> I'm not sure if this is even valid (probably not really?) but it doesn't seem
to have any purpose in any case and it triggers validation tests about the
two closing /li in a row.

Nested `<li>` elements are definitely invalid, so the "two closing /li in a row" error sounds like the browser is adding an implied end tag for the first li when the second one appears. Because React doesn't allow you to have unbalanced/mismatched tags like that, outside of `dangerouslySetInnerHTML`.